lib/constants.ts

```diff
@@ -36,7 +36,28 @@ export const INCOMING_SOUND_URL: string =
  * @private
  * Test names.
  */
-export enum TestNames {
+export enum TestName {
   InputAudioDevice = 'input-volume',
   OutputAudioDevice = 'output-volume',
 }
+
+/**
+ * All of the expected error names to be thrown by the diagnostics tests.
+ * These names are set in the error objects as the `.name` member.
+ */
+export enum ErrorName {
+  AlreadyStoppedError = 'already-stopped',
+  DiagnosticError = 'diagnostic',
+  InvalidOptionError = 'invalid-option',
+  InvalidOptionsError = 'invalid-options',
+  InvalidStateError = 'invalid-state',
+  PromiseTimedOutError = 'promise-timed-out',
+  UnsupportedError = 'unsupported',
+}
+
+/**
+ * All of the expected warnings to be thrown by the diagnostics tests.
+ */
+export enum WarningName {
+  LowAudioLevel = 'low-audio-level',
```

Comment:
Let's add a doc block here. We should explain when this warning is emitted, and when it's cleared. Let's mention the criteria for both.

Reply:
Check out client sdk docs on how we're explaining the warnings and the warnint-cleared criteria. I think the audio level is similar? https://www.twilio.com/docs/voice/insights/call-quality-events-twilio-client-sdk